building/toolchains.rst

```diff
@@ -64,10 +64,23 @@ paths and then you are ready to starting compiling OP-TEE components.
 
 LLVM / Clang
 ************
-It's possible to also compile :ref:`optee_os`.git using llvm/clang. Either get
-the toolchain using your package manager or alternatively build it yourself to
-get the version that you need. To build the llvm toolchain for use in OP-TEE do
-the following.
+It's possible to also compile :ref:`optee_os`.git using llvm/clang. To do that,
+you can download and extract Clang from the GitHub release page. You'll need an
+x86_64 cross-compiler capable of generating aarch64 and armv7a code **and** the
+compiler-rt libraries for these architectures (libclang_rt.*.a).
+
+Clang is configured to be able to cross-compile to all the supported
+architectures by default (see <clang path>/bin/llc --version) which is great,
+but compiler-rt is included only for the host architecture. Therefore you need
+to combine several packages into one. Please refer to this `get_clang.sh`_
+script for details on creating a llvm/clang toolchain ready to be used.
```

Comment:
Linking to the script is OK but we should mention `build.git` as we did for the GCC toolchain:
```
$ cd $HOME
$ git clone https://github.com/OP-TEE/build.git
$ cd build
$ make clang-toolchains
```
...downloads and installs Clang in `$HOME/clang-9.0.1`.